docker-compose.yml

```diff
@@ -1,18 +1,25 @@
-version: "3.8"
+version: "3"
```

Comment:
The version field is deprecated as per the compose specification now: https://github.com/compose-spec/compose-spec/blob/master/spec.md#version-top-level-element, maybe we could remove this now?

Reply:
I'm aware that it's deprecated, but older version of compose still need it, so for now, it's a good idea to keep it.

Reply:
Looks like you're right, version 1.28 works fine, however, 1.26 seems to choke saying "Unsupported config option for ..."